uv.lock

```diff
@@ -645,7 +645,7 @@ wheels = [
 
 [[package]]
 name = "craft-providers"
-version = "2.3.0"
+version = "2.4.0"
```

Comment:
I can revert this if that's preferrable; this happened because I was experimenting with a dev version of craft-providers